test/stream-framed.js

```diff
@@ -16,3 +16,28 @@ test('framed mode', function (t) {
   a.write(Buffer.of(0x2, 0x0, 0x0))
   a.write(Buffer.of(0x4, 0x5))
 })
+
+test('framed mode, large message', function (t) {
+  t.plan(1)
+
+  const [a, b] = makeTwoStreams(t, { framed: true })
+
+  const buf = Buffer.alloc(3 + 1024 * 4096 /* 4 MiB */)
+
+  buf[2] = 0x40
+
+  const recv = []
+
+  b
+    .on('data', (buffer) => {
+      recv.push(buffer)
+    })
+    .on('end', () => {
+      t.alike(Buffer.concat(recv), buf)
```

Comment:
unrelated, we should b4a all of these

Reply:
Fixed the two introduced in this PR, will open another for the rest.

Reply:
https://github.com/hyperswarm/libudx/pull/116